generator/autogenlist.ts

```diff
@@ -8,8 +8,224 @@ import { postProcessor as policyProcessor } from './processors/Microsoft.Authori
+    {
+        basePath: 'managedservices/resource-manager',
+        namespace: 'Microsoft.ManagedServices',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'marketplacenotifications/resource-manager',
+        namespace: 'Microsoft.MarketplaceNotifications',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'marketplaceordering/resource-manager',
+        namespace: 'Microsoft.MarketplaceOrdering',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'mediaservices/resource-manager',
+        namespace: 'Microsoft.Media',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'msi/resource-manager',
+        namespace: 'Microsoft.ManagedIdentity',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'network/resource-manager',
+        namespace: 'Microsoft.Network',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'operationsmanagement/resource-manager',
+        namespace: 'Microsoft.OperationsManagement',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'privatedns/resource-manager',
+        namespace: 'Microsoft.Network',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'providerhub/resource-manager',
+        namespace: 'Microsoft.ProviderHub',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'purview/resource-manager',
+        namespace: 'Microsoft.Purview',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'quota/resource-manager',
+        namespace: 'Microsoft.Quota',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'resources/resource-manager',
+        namespace: 'Microsoft.Features',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'resources/resource-manager',
+        namespace: 'Microsoft.Solutions',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'securityandcompliance/resource-manager',
+        namespace: 'Microsoft.SecurityAndCompliance',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'service-map/resource-manager',
+        namespace: 'Microsoft.OperationalInsights',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'servicefabricmanagedclusters/resource-manager',
+        namespace: 'Microsoft.ServiceFabricManagedClusters',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'testbase/resource-manager',
+        namespace: 'Microsoft.TestBase',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'trafficmanager/resource-manager',
+        namespace: 'Microsoft.Network',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'videoanalyzer/resource-manager',
+        namespace: 'Microsoft.Media',
+        disabledForAutogen: true,
+    },
+    {
+        basePath: 'webpubsub/resource-manager',
+        namespace: 'Microsoft.SignalRService',
+        disabledForAutogen: true,
+    },
+];
```

Comment:
This list of a snapshot of the `list-basepaths` output - to avoid changing the list of RPs currently opted in/out.